README.md

```diff
@@ -1 +1,23 @@
-# edge-search-srs
\ No newline at end of file
+# edge-search-srs
+
+Copyright (C) 2018-2019 The Open Library Foundation
+
```

Comment:
The date-range in the top of the README should just be "2021", as explained in https://dev.folio.org/guidelines/create-new-repo/

Reply:
> Additional comments:
> 
> As per https://dev.folio.org/guidelines/create-new-repo/
> the required personal data disclosure form is missing.
> 
> There is no GitHub Team access enabled.

Hi @dcrossleyau, thank you for so helpful notes. I fixed all of them. Could you please take a final look again to ensure that I did everything correctly?) Thank you

Reply:
This is a new module. So:
Copyright (C) 2021 The Open Library Foundation